src/ICeWall/doc/place_cell.md

```diff
@@ -0,0 +1,29 @@
+## place_cell
+### Synopsis
+```
+  % place_cell [-name <name>] \
+                     [-type <type>] \
+                     [-cell <library_cell>] \
+                     -location {(centre|origin) {x <value> y <value>} orient (R0|R90|R180|R270|MX|MY|MXR90|MYR90)} \
+                     [-inst_name <instance_name>]
```

Comment:
This does not match the command.
I think you should integrate the command directly into ICeWall and restore the command options like they are here. Otherwise commands end up with origin and centre as their locations and the user is stuck figuring these things out.

Reply:
@Colin-Holehouse based on the comment below, I think you can ignore this about integrating it into the rest of ICeWall.
But I would suggest adding a `-create/-add` to be explicit about wanting to create a new instance or use an existing one and maybe return the instance at the end?

Reply:
Updated documentation to match the command